teamcity: failed test: version-upgrade

The following tests appear to have failed on master (roachtest): acceptance/version-upgrade

You may want to check [for open issues](https://github.com/cockroachdb/cockroach/issues?q=is%3Aissue+is%3Aopen+version-upgrade).

[#1586440](https://teamcity.cockroachdb.com/viewLog.html?buildId=1586440):


```
acceptance/version-upgrade
--- FAIL: roachtest/acceptance/version-upgrade (178.507s)
test artifacts and logs in: artifacts/acceptance/version-upgrade/run_1
	upgrade.go:532,upgrade.go:589,acceptance.go:74,test_runner.go:697: pq: value differs between gossip (2.1) and KV (2.0); try again later (<nil> after 1m59.912258995s)

test artifacts and logs in: artifacts/acceptance/version-upgrade/run_1
	upgrade.go:532,upgrade.go:589,acceptance.go:74,test_runner.go:697: pq: value differs between gossip (2.1) and KV (2.0); try again later (<nil> after 1m59.912258995s)





```

Please assign, take a look and update the issue accordingly.

--- a/pkg/cmd/roachtest/acceptance.go
+++ b/pkg/cmd/roachtest/acceptance.go
@@ -44,7 +44,16 @@ func registerAcceptance(r *testRegistry) {
 		{name: "gossip/locality-address", fn: runCheckLocalityIPAddress},
 		{name: "rapid-restart", fn: runRapidRestart},
 		{name: "status-server", fn: runStatusServer},
-		{name: "version-upgrade", fn: runVersionUpgrade, minVersion: "v19.1.0", timeout: 30 * time.Minute},
+		{
+			name: "version-upgrade",
+			fn:   runVersionUpgrade,
+			// This test doesn't like running on old versions because it upgrades to
+			// the latest released version and then it tries to "head", where head is
+			// the cockroach binary built from the branch on which the test is
+			// running. If that branch corresponds to an older release, then upgrading
+			// to head after 19.2 fails.
+			minVersion: "v19.2.0",
+			timeout:    30 * time.Minute},
 	}
 	tags := []string{"default", "quick"}
 	const numNodes = 4

--- a/pkg/cmd/roachtest/upgrade.go
+++ b/pkg/cmd/roachtest/upgrade.go
@@ -477,8 +477,7 @@ func runVersionUpgrade(ctx context.Context, t *test, c *cluster) {
 		binaryVersionUpgrade("v19.1.5", nodes),
 		clusterVersionUpgrade(""),
 
-		// TODO(andrei): Change to the final 19.2 version once released.
-		binaryVersionUpgrade("v19.2.0-rc.2", nodes),
+		binaryVersionUpgrade("v19.2.1", nodes),
 		clusterVersionUpgrade(""),
 
 		// Each new release has to be added here. When adding a new release, you'll

